CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+2.0.0 (2015-XX-XX)
+---------------------
+- Default timeout in fido.fetch(..) has changed from 1s to None (wait indefinitely)
+  This will change the behavior of existing code that doesn't pass in a timeout
```

Comment:
It will be helpful to add the 'None' explanation (i.e. wait indefinitely) in timeout docstring in fetch() as well